functions.php

```diff
@@ -154,18 +154,17 @@ function genesis_sample_post_type_support() {
 /**
  * Reduces secondary navigation menu to one level depth.
  *
- * @since 2.2.3
+ * @since 3.1.0
```

Comment:
@seothemes The `@since` doesn't need to change here. (The function was still introduced in 2.2.3, and its name and functionality haven't changed.)